lib/index.js

```diff
@@ -45,9 +45,9 @@ function inscriptionModificationOpenForYoungs(cohort) {
     case "Février 2023 - C":
       return new Date() <= new Date(2023, 0, 12, 23, 59); // before 9 janvier 2023 23h59
     case "Avril 2023 - A":
-      return new Date() < new Date(2023, 1, 7); // before 7 fevrier 2023 morning
+      return new Date() < new Date(2023, 1, 7, 23, 59); // before 12 février 2023 23h59
     case "Avril 2023 - B":
-      return new Date() < new Date(2023, 1, 21); // before 21 fevrier 2023 morning
+      return new Date() < new Date(2023, 1, 28, 23, 59); // before 26 fevrier 2023 23h59
```

Comment:
Le 28 ? Soyons coherent !
```suggestion
      return new Date() < new Date(2023, 1, 28, 23, 59); // before 28 fevrier 2023 23h59
```

Reply:
Si je comprends bien, cette variable c'est le moment où les volontaires n'auront plus accès à leur formulaire d'inscription ? Si oui c'est bien le 28 et non le 26, qui est le jour de la fermeture de la possibilité de créer une nouvelle inscription, mais pas de modifier une existante.

Reply:
La on veux pas qu'il puisse modifier son dossier, mais l'inscription et la proposition de sejour est deja close